Formula/mrbayes.rb

```diff
@@ -12,31 +12,26 @@ class Mrbayes < Formula
     sha256 "de35643f5dc2c6f2234aed0d121b5446688fda285d0ecdd450a0258afc728717" => :x86_64_linux
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "beagle" => :recommended
-  depends_on "open-mpi" => :recommended
+  depends_on "beagle"   => :recommended
+  depends_on "readline" => :recommended
+  depends_on "open-mpi" => :optional
 
   def install
-    args = ["--disable-debug", "--prefix=#{prefix}"]
-    args << "--with-beagle=" + (build.with?("beagle") ? Formula["beagle"].opt_prefix : "no")
-    args << "--enable-mpi="  + (build.with?("open-mpi") ? "yes" : "no")
+    args = ["--prefix=#{prefix}"]
+    args << "--with-beagle="   + (build.with?("beagle")   ? Formula["beagle"].opt_prefix : "no")
+    args << "--with-mpi="      + (build.with?("open-mpi") ? "yes" : "no")
+    args << "--with-readline=" + (build.with?("readline") ? "yes" : "no")
 
-    cd "src" do
-      system "autoconf"
-      system "./configure", *args
-      system "make"
-      bin.install "mb"
-    end
-
-    pkgshare.install ["documentation", "examples"]
+    system "./configure", *args
+    system "make"
+    system "make", "install"
```

Comment:
```suggestion
    system "make", "install"
    pkgshare.install [share/"documentation", share/"examples"]
```
Could these files please be moved back to their previous location in `pkgshare`?